tests/epyccel/recognised_functions/test_epyccel_pyc_math.py

```diff
@@ -0,0 +1,95 @@
+# pylint: disable=missing-function-docstring, missing-module-docstring/
+# pylint: disable=wildcard-import
```

Comment:
Please do not disable wildcard import errors

Reply:
ok